### PR TITLE
add a partners page

### PR DESCRIPTION
**Pull request by DavidVorick**
to honor our new crypti partnership

### CODE REVIEW

File: public_html/partners.html [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+						Crypti provides a full stack solution for deploying truly decentralized applications onto the blockchain, while also providing an excellent user expereince for anyone looking to make safe and easy transactions within seconds.
+						Crypti's very own Dapp store gives consumers the same if not superior interface consumers have come to expect from existing centralized solutions currently dominating the market.
+						Each Dapp operates on a separate custom sidechain secured by Crypti's own version of the Delegated Proof of Stake consensus model.
+						For the progressive startup looking for the perfect platform on which to launch their first Dapp, Crypti is the number one choice.
+						Further development of the core protocol is maintained by the Crypti Foundation, a dedicated team of developers, entrepreneurs and engineers from across the globe.
+						<br><br>
+						Crypti Website: <a href="https://crypti.me">crypti.me</a><br>
+						Crypti Blog: <a href="https://blog.crypto.me">blog.crypti.me</a><br>
+						Crypti BTT Thread: <a href="https://https://bitcointalk.org/index.php?topic=654463.0">bitcointalk.org/index.php?topic=65446300</a><br>
```
Comment (agundy) [inline public_html/partners.html]: Extra `https://`?

Comment (DavidVorick) [inline public_html/partners.html]: ah yeah, thanks

Comment (triazo) [inline public_html/partners.html]: also the link above should be blog.crypti.me

**Comment by DavidVorick** [comment]
thanks guys

**Pull request merged**